packages/clarity-js/src/data/baseline.ts

```diff
@@ -61,9 +75,20 @@ export function track(event: Event, x: number, y: number, time?: number): void {
             buffer.scrollY = y;
             buffer.scrollTime = time;
             break;
+        case Event.MouseMove:
+            buffer.moveX = x;
+            buffer.moveY = y;
+            buffer.moveTime = time;
+        // intentionally skipping break to ensure pointer data is also updated for mouse move event
```

Comment:
this doesn't seem correct to me. On a mouseMove event we update the downX, downY and downTime?

Reply:
yes, i was on the fence about this. updated to copy the logic from default to two new blocks